playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -31,7 +31,7 @@
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.common.hostname | lower }} {{ openshift.common.evacuate_or_drain }} --force
+      {{ openshift.common.admin_binary }} drain {{ openshift.common.hostname }} --force --delete-local-data --ignore-daemonsets
```

Comment:
@sdodson any reason to use `admin_binary` instead of `client_binary` and adding `adm`?